docs/tutorials/monolayer.ipynb

```diff
@@ -237,20 +239,40 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "dspc = SurfactantLayer.from_pars(\n",
-    "    tail_layer_molecular_formula=tail_formula,\n",
-    "    tail_layer_thickness=tail_thickness,\n",
-    "    tail_layer_solvent=air,\n",
-    "    tail_layer_solvent_fraction=tail_solvent_fraction, \n",
-    "    tail_layer_area_per_molecule=area_per_molecule,\n",
-    "    tail_layer_roughness=roughness,\n",
-    "    head_layer_molecular_formula=head_formula,\n",
-    "    head_layer_thickness=head_thickness,\n",
-    "    head_layer_solvent=d2o,\n",
-    "    head_layer_solvent_fraction=head_solvent_fraction, \n",
-    "    head_layer_area_per_molecule=area_per_molecule,\n",
-    "    head_layer_roughness=roughness\n",
+    "tail_layer = LayerAreaPerMolecule(\n",
+    "    molecular_formula=tail_formula,\n",
+    "    thickness=tail_thickness,\n",
+    "    solvent=air,\n",
+    "    solvent_fraction=tail_solvent_fraction, \n",
+    "    area_per_molecule=area_per_molecule,\n",
+    "    roughness=roughness\n",
     ")\n",
+    "head_layer = LayerAreaPerMolecule(\n",
+    "    molecular_formula=head_formula,\n",
+    "    thickness=head_thickness,\n",
+    "    solvent=d2o,\n",
+    "    solvent_fraction=head_solvent_fraction, \n",
+    "    area_per_molecule=area_per_molecule,\n",
+    "    roughness=roughness\n",
+    ")\n",
+    "dspc = SurfactantLayer(\n",
+    "    tail_layer=tail_layer,\n",
+    "    head_layer=head_layer\n",
+    ")\n",
+    "# dspc = SurfactantLayer.from_pars(\n",
```

Comment:
unless needed, please remove the commented out code.

Reply:
done